templates/network-plugins/cilium/cilium-vxlan.yaml.tpl

```diff
@@ -746,16 +664,21 @@ spec:
   template:
     metadata:
       annotations:
-        prometheus.io/port: "9962"
-        prometheus.io/scrape: "true"
+        # Set app AppArmor's profile to "unconfined". The value of this annotation
+        # can be modified as long users know which profiles they have available
+        # in AppArmor.
+        container.apparmor.security.beta.kubernetes.io/cilium-agent: "unconfined"
+        container.apparmor.security.beta.kubernetes.io/clean-cilium-state: "unconfined"
+        container.apparmor.security.beta.kubernetes.io/mount-cgroup: "unconfined"
+        container.apparmor.security.beta.kubernetes.io/apply-sysctl-overwrites: "unconfined"
       labels:
         k8s-app: cilium
         app.kubernetes.io/name: cilium-agent
         app.kubernetes.io/part-of: cilium
     spec:
       containers:
       - name: cilium-agent
-        image: "${agent_repo}:${agent_tag}"
+        image: "quay.io/cilium/cilium:v1.14.2"
```

Comment:
Hi @imo-ininder 
I found that image version changes have been applied on `variables_defaults.tf`.
I'd think we should keep `"${agent_repo}:${agent_tag}"` if it works.